# Points should be in gait-coordinates:
# x: forward
# y: upward
# z: sideways
# midpoint_deviaiton = 0.05 + min(0.5*(y-0.05), 0.15)
# midpoint_height = 0.15 + max(y, 0)
points:
  - x: 0.4
    y: 0.0
    z: 0.45
    duration: 1.3
    midpoint_deviation: 0.05
    midpoint_height: 0.15
  - x: 0.4
    y: 0.1
    z: 0.45
    duration: 1.8
    midpoint_deviation: 0.075
    midpoint_height: 0.16
  - x: 0.4
    y: 0.0
    z: 0.45
    duration: 1.8
    midpoint_deviation: 0.05
    midpoint_height: 0.15
  - x: 0.4
    y: 0.0
    z: 0.45
    duration: 1.3
    midpoint_deviation: 0.05
    midpoint_height: 0.15
  - x: 0.4
    y: -0.1
    z: 0.45
    duration: 1.8
    midpoint_deviation: 0.075
    midpoint_height: 0.15
  - x: 0.4
    y: 0.0
    z: 0.45
    duration: 1.8
    midpoint_deviation: 0.05
    midpoint_height: 0.15